phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -2387,8 +2452,27 @@ public Void call() throws Exception {
                                     logger.info("Update of SYSTEM.CATALOG complete");
 									clearCache();
                                 }
-                                
+
+                                if (currentServerSideTableTimeStamp < MetaDataProtocol.MIN_SYSTEM_TABLE_TIMESTAMP_4_8_0) {
+                                    // Add these columns one at a time, each with different timestamps so that if folks
```

Comment:
This comment is just a copy paste of the comment in the previous if block. It would make more sense to move this comment to above if (currentServerSideTableTimeStamp < MetaDataProtocol.MIN_SYSTEM_TABLE_TIMESTAMP_4_6_0) {} block as it applies to all the upgrades we will do in future.